rocAL/source/parameters/parameter_factory.cpp

```diff
@@ -104,33 +104,43 @@ void ParameterFactory::generate_seed() {
     _seed = rd();
 }
 
+int64_t
+ParameterFactory::get_seed_from_seedsequence() {
```

Comment:
Please move the function definition to previous line to make it inline with return type
```
int64_t ParameterFactory::get_seed_from_seedsequence() {
```